categories/projects.ts

```diff
@@ -0,0 +1,8 @@
+import { category } from "../builder";
+
+const projects = category({
+    ...(await import("./projects/c2c")).default,
+    ...(await import("./projects/featurePerception")).default,
```

Comment:
Really simple method for import definition -- and works with type safety!!

This could be 'automated' / all the imports could be code generated (perhaps when the development server is running)